app/views/posts/index.html.erb

```diff
@@ -1,3 +1,23 @@
+<div class="flex mt-2">
+        <img src=<%= @user.Photo %> >
+        <div class="flex justify-between flex1">
+            <div class="user"><%= @user.Name %></div>
+            <div class="number">Number of posts: <%= @user.PostsCounter %></div>
+        </div>
+</div>
 <div>
-  This is a list of posts by one user
+<% @posts.each do |post| %>
+    <div class="mt-2">
+        <div><strong> <%= post.Title %> </strong></div>
+        <div> <%= post.Text %> </div>
+        <div class="align-left"> Comments: <%= post.CommentsCounter %>  Likes: <%= post.LikesCounter %> </div>
+        <div>
```

Comment:
- [ ] Please make sure clicking on each post takes the user to the post-show page.